sdk/eventhub/event-hubs/src/dataTransformer.ts

```diff
@@ -24,14 +24,15 @@ export const defaultDataTransformer = {
    * - multiple: true | undefined.
    */
   encode(body: any): any {
+    // eslint-disable-line @typescript-eslint/explicit-module-boundary-types
     let result: any;
```

Comment:
can we change `any` to `unknown` in the argument type throughout to fix the linting the issue? it could be a bit painful sometimes but it is safer to work with.

Reply:
Another option is to refactor the if/else block to return early so that we don't need to have the `result` variable at all.

```js
if (isBuffer(body) {
     return message.data_section(body);
}

try {
   const bodyStr = JSON.stringify(body);
   return message.data_section(Buffer.from(bodyStr, "utf8"));
} catch (err) {

}

Reply:
@ramya-rao-a sorry for the confusion by putting my comment on line 28 instead of line 27. I meant to do this change for arguments only (`body` in this case).

Reply:
Removed all of the `@typescript-eslint/explicit-module-boundary-types` overrides in #13285